### PR TITLE
Installer args

### PR DESCRIPTION
**Pull request by michaelcdillon**
Added the start menu shortcut page back into the installer. Also setup embedded Bittorrent installer call to use force install args and it reflects the user's choice on the start menu shortcut page.

### CODE REVIEW

File: project/Win32BuildSetup/BuildSetup.bat [hunk 1]
```diff
@@ -50,21 +51,33 @@ SET buildbinaryaddons=true
 SET exitcode=0
 SET useshell=rxvt
 SET BRANCH=na
+SET INSTALLER_ONLY=false
+SET EXE_AND_INSTALLER=false
 FOR %%b in (%1, %2, %3, %4, %5, %6) DO (
```
Comment (jjoyce-bittorrent) [inline project/Win32BuildSetup/BuildSetup.bat]: this should have a ,%7, %8 I think

File: project/Win32BuildSetup/BuildSetup.bat [hunk 2]
```diff
@@ -50,21 +51,33 @@ SET buildbinaryaddons=true
...
 
 
   :: sets the BRANCH env var
   call getbranch.bat
+  IF %INSTALLER_ONLY%==true (
+	ECHO "Running Installer Build Only"
+	GOTO NSIS_EXE
+  )
+  
+  IF %EXE_AND_INSTALLER%==true (
```
Comment (jjoyce-bittorrent) [inline project/Win32BuildSetup/BuildSetup.bat:77]: how is this different from the build option 'nomingwlibs'?

File: project/Win32BuildSetup/BuildSetup.bat (hunk 1 above)
Comment (michaelcdillon) [reply project/Win32BuildSetup/BuildSetup.bat]: That would be true if we wanted to support all the different options at the same time. I don't mind putting it in though.

File: project/Win32BuildSetup/BuildSetup.bat (hunk 2 above)
Comment (michaelcdillon) [reply project/Win32BuildSetup/BuildSetup.bat:77]: Its been a while, but I'm pretty sure this is faster than `nomingwlibs`.

**Review by jjoyce-bittorrent — APPROVED** [review]
lgtm

File: project/Win32BuildSetup/BuildSetup.bat (hunk 1 above)
Comment (michaelcdillon) [reply project/Win32BuildSetup/BuildSetup.bat]: I addressed this.

**Pull request merged**